### DISCUSSION TITLE
官网那样的评论样式怎么改啊

### DISCUSSION THREAD

**Discussion by OrdChaos** [walinejs/waline#874] [Q&A] [answered]
如题

**Comment by Mister-Hope** [comment, marked as answer]
啥叫官网那样的评论样式

**Reply by OrdChaos** [reply]
就是waline.is.org文档每个页面的Demo，感觉比默认的好多了

**Reply by Mister-Hope** [reply]
没改动任何样式。。。。

**Reply by OrdChaos** [reply]
肯定改过了，至少按钮颜色不一样

**Reply by Mister-Hope** [reply]
。。。。。。

https://waline.js.org/guide/client/style.html

https://github.com/walinejs/waline/blob/main/docs/src/.vuepress/styles/index.scss#L2
